src/css/components/_c-person.scss

```diff
@@ -107,11 +107,22 @@ $_person-distance: (
 	@media screen and (prefers-reduced-motion: reduce) {
 		transition: none;
 	}
+
+	@media (prefers-color-scheme: dark) {
+		background-color: $color-black;
+		border-color: $color-gray-shade-darkmode;
+
+		// States
+		&:hover {
+			background-color: $color-black;
+			border-color: $color-gray-tint-darkmode;
+		}
+	}
 }
 
 
 .c-person__link {
-	@include link-states($color-blue-shade);
+	@include link-states($color-blue-shade-lightmode);
```

Comment:
$color-blue-shade-lightmode seems to return a color hex and not color name. Hence this results in `--color-#232d71` which gives an error and doesn't render correctly. Can be seen in team page. Same for the dark mode.